### PR TITLE
Update Results Card Data Points

### PR DESCRIPTION
**Pull request by Icemanbeta**
- Swap **COLA** and **Danger Pay**
- Update **View Position** -> **View details**
- Titlecase position title

### CODE REVIEW

File: src/Components/ResultsCard/ResultsCard.jsx [hunk 1]
```diff
@@ -75,8 +76,8 @@ const ResultsCard = (props) => {
       'Tour of Duty': getResult('post.tour_of_duty', NO_TOUR_OF_DUTY),
       'Language': language,
       'Post Differential': getResult('post.differential_rate', NO_POST_DIFFERENTIAL, true),
-      'COLA': getResult('post.cost_of_living_adjustment', NO_COLA, true),
       'Danger Pay': getResult('post.danger_pay', NO_DANGER_PAY, true),
+      'COLA': getResult('post.cost_of_living_adjustment', NO_COLA, true),
```
Comment (burgwyn) [inline src/Components/ResultsCard/ResultsCard.jsx]: 🚫🥤

File: src/Components/ResultsCard/ResultsCard.jsx [hunk 2]
```diff
@@ -108,8 +109,8 @@ const ResultsCard = (props) => {
         <div id={result.id} className="results-card">
           <Row className="header" fluid>
             <Column columns="6">
-              <h3>{result.title}</h3>
-              <Link to={`/details/${result.position_number}`}>View Position</Link>
+              <h3>{title}</h3>
+              <Link to={`/details/${result.position_number}`}>View details</Link>
```
Comment (burgwyn) [inline src/Components/ResultsCard/ResultsCard.jsx]: Change the label to View Position

Comment (Icemanbeta) [reply src/Components/ResultsCard/ResultsCard.jsx]: @burgwyn According to https://github.com/18f/state-talentmap/issues/1686#issuecomment-376616690, **View details** is the latest text version for the link

File: src/Components/ResultsCard/ResultsCard.jsx (hunk 1 above)
Comment (Icemanbeta) [reply src/Components/ResultsCard/ResultsCard.jsx]: 🚫🥤 = No cola? lol

File: src/Components/ResultsCard/ResultsCard.jsx (hunk 2 above)
Comment (Icemanbeta) [reply src/Components/ResultsCard/ResultsCard.jsx]: PS I even changed the compare button back to the checkbox, if that design is out of date 😭

Comment (mjoyce91) [reply src/Components/ResultsCard/ResultsCard.jsx]: I think that's out of date since some recent updates :sweat_smile:

Should be "View Position"

**Comment by Icemanbeta** [comment]
This turned into just removing **COLA** then lol

**Pull request merged**